addons/bestja_offers/templates.xml

```diff
@@ -11,39 +11,49 @@
                     <div class="cover">
                         <section class="row">
                             <section class="offer_name col-md-3 col-md-offset-1 col-xs-7 col-xs-offset-1">
-                                <h1><t t-esc="offer.name"/></h1>
-
-                                <p class="offer_organization"><t t-esc="offer.project.organization.name"/>
-
-                                </p>
+                                <h1>
+                                    <t t-esc="offer.name"/>
+                                </h1>
 
                             </section>
 
-                            <section class="offer_share col-md-2">
-                            <t t-call="bestja_offers.share" />
+                            <section class="col-md-2">
+                            
                             </section>
 
-                            <section class="offer_icons col-md-3 col-md-offset-2 col-xs-4 col-xs-offset-0">
-                                <div class="offer_icon_container"><div class="offer_icon">ico</div></div>
-                                <div class="offer_icon_container"><div class="offer_icon">ico</div></div>
-                                <div class="offer_icon_container"><div class="offer_icon">ico</div></div>
+                            <section class="offer_share col-md-3 col-md-offset-2 col-xs-4 col-xs-offset-0">
+                                <t t-call="bestja_offers.share" />
                             </section>
                         </section>
                     </div>
 
                     <section class="row offer_info">
                         <section class="col-md-3">
-                            <h2>Gdzie</h2>
-                            <p><t t-esc="offer.city"/> <t t-esc="offer.district"/></p>
+                            <h3>Termin</h3>
+                            <h2>od <span t-field="offer.project.date_start" t-field-options='{"format": "d.MM.y"}'/> 
+                                <br />
+                                do <span t-field="offer.project.date_stop" t-field-options='{"format": "d.MM.y"}'/>
+                            </h2>
+                        </section>
+                        <section class="col-md-3">
+                            <h3>Gdzie</h3>
+                            <h2>
+                                <t t-esc="offer.city"/> 
+                                <t t-esc="offer.district"/>
+                            </h2>
                         </section>
                         <section class="col-md-3">
-                            <h2>Liczba godzin</h2>
-                            <p><t t-esc="offer.hours"/> / tyg.</p>
+                            <h3>Kiedy</h3>
+                            <h2 t-if="offer.hours">
```

Comment:
You use headers in a very non-semantic way (in various places). Headers are for... headers. If you need a bigger text just make up a css class.